server/src/mindustry/server/ServerControl.java

```diff
@@ -56,19 +56,53 @@ public class ServerControl implements ApplicationListener{
         }
     };
 
-    private Fi currentLogFile;
-    private boolean inGameOverWait;
-    private Task lastTask;
-    private Gamemode lastMode;
-    private @Nullable Map nextMapOverride;
-    private Interval autosaveCount = new Interval();
+    /** The file to which the logs are currently being written. */
+    public Fi currentLogFile;
+
+    /** Whether the server is currently waiting for the next map to be loaded. */
+    public boolean inGameOverWait;
+
+    /** The current timer for loading the next map. */
+    public Task lastTask;
```

Comment:
This *still* should not be public, and this documentation is lacking. If you need a method to cancel it, add that, but keep it private.